src/EventHandler/Filter/FilterCallbackRegex.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
+
+namespace danog\MadelineProto\EventHandler\Filter;
+
+use Attribute;
+use danog\MadelineProto\EventHandler\CallbackQuery;
+use danog\MadelineProto\EventHandler\Update;
+use Webmozart\Assert\Assert;
+
+#[Attribute(Attribute::TARGET_METHOD)]
+class FilterCallbackRegex extends Filter
```

Comment:
LGTM, but I'm not too convinced this is needed :)

Reply:
you mean i remove entire FilterCallbackRegex in this case?! why?
we have talk about this in telegram and i have example use of this pattern its not useless